lib/evm_watchdog.rb

```diff
@@ -1,95 +0,0 @@
-require 'time'
-require 'linux_admin'
-require 'fileutils'
-
-module EvmWatchdog
-  PID_LOCATION = '/var/www/miq/vmdb/tmp/pids/evm.pid'.freeze
-  UPDATE_FILE  = '/var/www/miq/vmdb/tmp/miq_update'.freeze
-
-  def self.check_evm
-    pid_file = read_pid_file(PID_LOCATION)
-    pid_ps = get_ps_pids('MIQ Server')
-    if pid_file.nil?
-      # EVM exited gracefully - no pid file, nothing to do.
-      # Future TODO - Check database to see if EVM should be started?
-    elsif pid_file.empty? # Not really sure how we got an empty pid file, but we'll just log it for now.
-      log_info("Detected an empty PID file for EVM Server Process.")
-    elsif pid_file.include? "no_db"
-      db_state = get_db_state # TODO: - This is here because it is costly.  Should be moved up when not as costly.
-      if db_state.empty?
```

Comment:
not sure if we can put in a dependency on the database so evm server restarts when the db comes back up?